src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,478 @@
+        '403':
+          description: Delete not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v1/credential/{credentialId}/suspend:
+    patch:
+      tags:
+        - Credential
+      summary: (Un)suspend a credential
+      operationId: suspendCredential, unsuspendCredential
+      parameters:
+        - name: credentialId
+          in: path
+          description: the id of the credential
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SuspendCredential'
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+components:
+  responses:
+    CredentialRequestResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/CredentialRequestId'
+              - $ref: '#/components/schemas/PersonId'
+              - $ref: '#/components/schemas/CredentialRequest'      
+
+    CredentialResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/CredentialId'
+              - $ref: '#/components/schemas/PersonId'
+              - $ref: '#/components/schemas/Credential'
+
+  schemas:
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          description: Error code
+          type: integer
+          format: int32
+        message:
+          description: Error message
+          type: string
+
+    PersonId:
+        type: object
+        properties:
+          personId:
+            type: string
+            description: The unique id of person that the credential request is for
+
+    CredentialRequestId:
+      type: object
+      properties:
+        credentialRequestId:
+          type: string
+          description: The unique id of this credential request
+
+    CredentialRequest:
+      description: A request for a credential
+      type: object
+      properties:
+        credentialProfileId:
```

Comment:
where can I find more info on credential profiles?
what's the structure of a credential profile?